framework/db/oci/Schema.php

```diff
@@ -57,7 +58,25 @@ public function releaseSavepoint($name)
      */
     public function quoteSimpleTableName($name)
     {
-        return strpos($name, '"') !== false ? $name : '"' . $name . '"';
+        if (strpos($name, '"') !== false) {
+            return $name;
+        } elseif ($this->db->slavePdo->getAttribute(PDO::ATTR_CASE) === PDO::CASE_LOWER) {
+            $name = strtoupper($name);
```

Comment:
Why?
This whould NOT work, if tabled named in "CamelCase", for example: 'UserComment'.